matter/wifi/wfx_host_events.h

```diff
@@ -47,11 +47,11 @@
 #include "wfx_msgs.h"
```

Comment:
The file has a lot of defines. It would be worth while to break them down in section with comments on their purpose and use.
Making sure they are all necessary would be a good idea as the same time

Reply:
done